lib/autobuild/package.rb

```diff
@@ -68,6 +68,10 @@ def import=(value)
         # @return [Array<EnvOp>]
         attr_reader :env
 
+        #attr_accessor :prepared
```

Comment:
Leftover ?

I'm phasing out of my style of using accessors for everything. Please use the `@ivar` form and only define the predicate accessors (that you've already defined)